src/content/docs/en/pages/guides/marketplace/integrations/how-to-massive-redirection/index.mdx

```diff
@@ -1,55 +1,58 @@
+This integration permits you to control this domain redirection directly at the network's edge, centralizing the management of several settings in one location. It also enables you to operate a variety of situations, such as CMS or e-commerce platform migration, website upgrades, and many others. No matter how many pages you have, you can configure all your redirects with the Massive Redirect integration.
 
-> **Important**: you have to observe the limit of 300kb for the JSON file.
+:::note
+You have to observe the limit of 300kb for the JSON file.
+:::
 
 ---
 
-## Getting the solution
+## Getting the integration
 
-To use the Massive Redirect solution provided by Azion Marketplace, you have to:
+To use the **Massive Redirect** integration provided by Azion Marketplace, you have to:
 
-1. Log in to your [Real-Time Manager (RTM)](https://manager.azion.com/) account.
-    - If you don't have one yet, [create an account visiting the sign-up page](https://manager.azion.com/signup/).
-2. On the upper-left corner of the page, select **Products menu** > **Marketplace**.
-3. Select the search box and type "*massive*" or browse through the cards to find the Massive Redirect solution.
-4. Once you've found the Massive Redirect card, select it to go the solution page.
-5. On the solution page, look for the **Subscribe for** section on the bottom-right corner.
-6. Click the **Get It Now** button.
+1. Access [Real-Time Manager (RTM)](/en/documentation/products/guides/how-to-access-rtm/) > **Marketplace**.
+2. On the Marketplace homepage, select the integration's card.
+3. Once the integration's page opens, click the **Get It Now** button, at the bottom-right corner of the page.
 
-> You'll see a message indicating that your solution was successfully installed and is ready to use.
+You'll see a message indicating that your integration was successfully installed.
 
-## Using the solution
+:::tip
+You can search any integration by browsing through the cards or typing a keyword in the search bar.
+:::
 
-Once you've gotten your solution on **Marketplace**, it'll be available at your edge functions list. To use the solution, you'll have to create a new edge application.
+---
+
+## Configuring the integration
+
+Once you've gotten your integration on **Marketplace**, it'll be available at your edge functions list. To use the integration, you'll have to create a new edge application.
 
-To create an edge application you have three options:
+:::tip
+To create an edge application, go to the [building](/en/documentation/products/guides/build/build-an-application/) section for more details.
```

Comment:
```suggestion
To create an edge application, go to the [building](/en/documentation/products/guides/build/build-an-application/) section.
```

Reply:
achei meio desnecessário o final, acho que principalmente pelo verbo ser "to create". se fosse um "to find out more" acho que faria mais sentido ter o more details